account_move_tier_validation/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+# Copyright <2020> PESOL <info@pesol.es>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+{
+    "name": "Account Move Tier Validation",
+    "summary": "Extends the functionality of Account Moves to "
+    "support a tier validation process.",
+    "version": "13.0.1.0.0",
+    "category": "Accounts",
+    "website": "https://github.com/OCA/purchase-workflow",
+    "contributors": ["Odoo Peru"],
```

Comment:
Many not need this. Contributors goes to readme/CONTRIBUTORS.rst already.
But you can add yourself as maintainer instead :)
```
"maintainers": ["<your git account>"],
```

Reply:
removed contributors in last commit